packages/flutter/lib/src/services/asset_bundle.dart

```diff
@@ -198,7 +198,7 @@ abstract class CachingAssetBundle extends AssetBundle {
     }
     // The code above hasn't yet run its "then" handler yet. Let's prepare a
     // completer for it to use when it does run.
-    completer = new Completer<dynamic>();
+    completer = new Completer<T>();
     _structuredDataCache[key] = completer.future;
```

Comment:
We can't use T here because the compiler doesn't reify it.

Reply:
oh I guess when it doesn't it just treats it as dynamic though? So it'll do the same thing? Ok.

Reply:
Yep. Even though in Dart 1 mode VM pipeline does not reify `T` it still parses the whole thing correctly and substitutes `dynamic` here.